scripts/list-sdk-components.sh

```
@@ -0,0 +1,75 @@
+DIRECTORY_PATH=$2
+OUTPUT_HTML_FILE_NAME=$3
+OUTPUT_HTML_DIR_PATH=$4
+COMPONENT_NAME=$5
+ZIP_FILE_NAME=$6
+
+OUTPUT_HTML_FILE_PATH="$OUTPUT_HTML_DIR_PATH/$OUTPUT_HTML_FILE_NAME"
+
+# List of files to exclude
+# Exclude the ZIP file and the output HTML file as they'll be
+# in the same directory as the other files
+EXCLUDED_FILES=($ZIP_FILE_NAME $OUTPUT_HTML_FILE)
```

Shellcheck

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)